bindata/assets/alerts/cpu-utilization-sno.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
I have noticed that we already have `cpu-utilization.yaml` alert [defined](https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/bindata/assets/alerts/cpu-utilization.yaml).
Is there a way we could reuse it instead of creating a new one ?

Reply:
I have also noticed that `cpu-utilization.yaml` is not used by this operator, is that correct ?
I have also noticed that you [changed](https://github.com/openshift/cluster-kube-apiserver-operator/commit/29d75eafb79519cf468c6bbc8b4568ed7468be29) the alert to describe the behaviour on an SNO cluster.

I'm confused, why do we need `2` separate alerts ?
/cc @dgrisonnet

Reply:
I'm not sure if we can, because before we merge this PR we need to make sure that `cpu-utilization.yaml` will not be created on an SNO cluster. Also, after this change your controller will be reconciling the alter only on SNO clusters. You can work on both PRs at the same time.